pkg/pillar/diskmetrics/diskmetrics.go

```diff
@@ -70,14 +70,15 @@ func CreateImg(ctx context.Context, log *base.LogObject, diskfile string, format
 	return nil
 }
 
-//ConvertImg do conversion of diskfile to outputFile with defined format
-func ConvertImg(ctx context.Context, log *base.LogObject, diskfile, outputFile, outputFormat string) error {
+//RolloutImgToBlock do conversion of diskfile to outputFile with defined format
+func RolloutImgToBlock(ctx context.Context, log *base.LogObject, diskfile, outputFile, outputFormat string) error {
 	if _, err := os.Stat(diskfile); err != nil {
 		return err
 	}
 	// writeback cache instead of default unsafe, out of order enabled, skip file creation
-	args := []string{"convert", "-t", "writeback", "-W", "-n", "-O", outputFormat, diskfile, outputFile}
-	output, err := base.Exec(log, "/usr/bin/qemu-img", args...).WithContext(ctx).CombinedOutput()
+	// Timeout 2 hours
+	args := []string{"convert", "--target-is-zero", "-t", "writeback", "-W", "-n", "-O", outputFormat, diskfile, outputFile}
```

Comment:
Is this change related to spec.sh update? Shouldn't this be a separate PR? If not, could you please shortly document (with PR description and/or as a code comment) why we need this change (for those like me who miss the context)?

Reply:
@milan-zededa You are right this change is not supposed to be there. Let me see what happened. Depending on the case, I might redo the PR.

Reply:
@jsfakian I think this came from another PR which was just merged. The way I ensure clean PRs is to only do
 git fetch upstream master
 git rebase -i upstream/master
where upstream is lf-edge/eve